docs/configuring-the-deploy-cli.md

```diff
@@ -0,0 +1,135 @@
+# Configuring the Deploy CLI
+
+Configuring the Deploy’s CLI is essential for establishing Auth0 credentials as well as generally modifying the behavior of the tool to your specific needs. There are two ways the Deploy CLI can be configured:
+
+- Configuration file (`config.json`)
+- Environment variables
+
+## Configuration file
+
+A standalone JSON file can be used to configure Deploy CLI. This file will usually reside in the root directory of your project and be called `config.json`.
+
+Example **config.json** file:
+
+```json
+{
+  "AUTH0_DOMAIN": "<YOUR_TENANT_DOMAIN>",
+  "AUTH0_CLIENT_ID": "<YOUR_CLIENT_ID>",
+  "AUTH0_ALLOW_DELETE": false
+}
+```
+
+> ⚠️ **NOTE:** Hard-coding credentials is not recommended, and risks secret leakage should this file ever be committed to a public version control system. Instead, passing credentials as [environment variables](#) is considered best practice.
+
+### Environment variables
+
+By default, the Deploy CLI ingests environment variables, providing the ability to pass credentials and other configurations to the tool without needing to publish to the `config.json` file. Environment variables can either be used to augment the `config.json` file or replace it altogether depending on the project needs.
+
+Non-primitive configuration values like `AUTH0_KEYWORD_MAPPING` and `AUTH0_EXCLUDED` can also be passed in through environment variables so long as these values are properly serialized JSON.
+
+To **disable** the consumption of environment variables for either the `import` or `export` commands, pass the `--env=false` argument.
+
+#### Examples:
+
+```shell
+# Deploying configuration for YAML formats without a config.json file
+AUTH0_DOMAIN=<YOUR_AUTH0_DOMAIN> \
+AUTH0_CLIENT_ID=<YOUR_CLIENT_ID> \
+AUTH0_CLIENT_SECRET=<YOUR_CLIENT_SECRET> \
+a0deploy import --input_file=local/tenant.yaml
```

Comment:
```suggestion
export AUTH0_DOMAIN=<YOUR_AUTH0_DOMAIN>
export AUTH0_CLIENT_ID=<YOUR_CLIENT_ID> 
export AUTH0_CLIENT_SECRET=<YOUR_CLIENT_SECRET> 

a0deploy import --input_file=local/tenant.yaml
```

Reply:
Just a suggestion, your example also works on having them run as a full command, but I personally find this suggestion a bit easier to understand.